src/LayerEditor/ui/data/regions_model.py

```diff
@@ -84,9 +85,9 @@ def get_region_names(self):
 
     def is_region_used(self, reg):
```

Comment:
Should not be in this class since it does not contain that information.

Reply:
Already moved to LEModel.